common/djangoapps/util/views.py

```diff
@@ -289,48 +289,28 @@ def _record_feedback_in_zendesk(
         custom_fields=None
 ):
     """
-    Create a new user-requested Zendesk ticket or send an email to Support Team.
```

Comment:
We had modified this file for fincancial_assistance_form. Now we have removed all custom changes in it.